aws/errors.go

```diff
@@ -25,6 +27,16 @@ func isNotFoundError(notFoundErrors []string) plugin.ErrorPredicateWithContext {
 				}
 			}
 		}
+
+		var ae smithy.APIError
```

Comment:
To avoid potentially changing v1 table error handling, how much effort would it be to create new versions of functions we use in `errors.go` in the `errors_v2.go` function, and then have the updated v2 tables use that function instead?